client/components/wordpress-logo/docs/example.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import WordPressLogo from 'components/wordpress-logo';
+import Card from 'components/card';
+
+module.exports = React.createClass( {
+	displayName: 'WordPressLogo',
+
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/design/wordpress-logo">WordPress Logo</a>
+				</h2>
+				<Card>
```

Comment:
Well, it's more than that. If you intend to use the WP logo for "My Sites", then use the gridicon at the preset sizes. If you intend to use the WP logo for branding, i.e. WordPress.com, **don't** use the gridicon.